File: generator.py
def generate_wordlist(name, dob, pet, company):
    words = []

    base_words = [name, dob, pet, company]

    for word in base_words:
        words.append(word)
        words.append(word.lower())
        words.append(word.upper())
        words.append(word + "123")
        words.append(word + "@123")

    for w1 in base_words:
        for w2 in base_words:
            if w1 != w2:
                words.append(w1 + w2)
                words.append(w1 + "@" + w2)
                words.append(w1 + "123" + w2)

    with open("wordlist.txt", "w") as f:
        for word in set(words):
            f.write(word + "\n")

    print("Wordlist generated successfully!")

File: generator.php
<?php
error_reporting(E_ALL);
ini_set('display_errors', 1);

// Show current directory (IMPORTANT DEBUG)
echo "<br>Current Directory: " . __DIR__ . "<br>";

if ($_SERVER["REQUEST_METHOD"] == "POST") {

    $name = $_POST['name'] ?? '';
    $dob = $_POST['dob'] ?? '';
    $pet = $_POST['pet'] ?? '';
    $company = $_POST['company'] ?? '';

    if ($name == "") {
        echo "❌ Name is required";
        exit();
    }

    $words = [];
    $base = [$name, $dob, $pet, $company];

    foreach ($base as $w) {
        if (!empty($w)) {
            $words[] = $w;
            $words[] = strtolower($w);
            $words[] = strtoupper($w);
            $words[] = $w . "123";
        }
    }

    // 🔥 FORCE FILE PATH (ABSOLUTE)
    $file_path = __DIR__ . "/wordlist.txt";

    $result = file_put_contents($file_path, implode("\n", $words));

    if ($result !== false) {
        echo "✅ FILE CREATED SUCCESSFULLY<br>";
        echo "📄 Location: " . $file_path;
    } else {
        echo "❌ FAILED TO CREATE FILE";
    }

} else {
    echo "⚠️ Not a POST request";
}
?>